PathFind.cpp

```diff
@@ -1,106 +1,106 @@
-// Copyright 2024 Aditya Behrani
-#include <fstream>
-#include <iostream>
-#include <queue>
-#include <vector>
-#include "src/graph.cpp"
-#include "src/bfs.cpp"
-#include "src/dfs.cpp"
-#include "src/dijkstra.cpp"
-
-int START, GOAL;
-
-Graph* buildGraph(std::string filename) {
-    Graph *g = new Graph();
-
-    std::fstream infile(filename);
-    if (!infile) {
-        std::cerr << "Unable to open " << filename << "\n";
-        return NULL;
-    }
-
-    int N;
-    infile >> N;
-    for (int i = 0; i < N; i++) {
-        g->addNode(i);
```

Comment:
why not make the size of the graph part of the constructor and initialize the nodes internally, instead of repeatedly calling a method to build it out from externally?